api/src/feeds/impl/datasets_api_impl.py

```diff
@@ -40,29 +40,18 @@ class DatasetsApiImpl(BaseDatasetsApi):
 
     @staticmethod
     def create_dataset_query():
-        return (
-            Query(
-                [
-                    Gtfsdataset,
-                    t_componentgtfsdataset.c["component"],
-                    Gtfsdataset.bounding_box.ST_AsGeoJSON(),
-                    Feed.stable_id,
-                ]
-            )
-            .join(
-                t_componentgtfsdataset,
-                t_componentgtfsdataset.c["dataset_id"] == Gtfsdataset.id,
-                isouter=True,
-            )
-            .join(Feed, Feed.id == Gtfsdataset.feed_id)
-        )
+        return Query(
+            [
+                Gtfsdataset,
+                Gtfsdataset.bounding_box.ST_AsGeoJSON(),
+                Feed.stable_id,
+            ]
+        ).join(Feed, Feed.id == Gtfsdataset.feed_id)
 
     @staticmethod
     def _load_validation_report():
```

Comment:
I've noticed that this method is responsible for loading the notices. Given that we work with historical datasets, which include all report versions for all datasets, the volume of data retrieved could become significantly large. To optimize our data handling and focus on relevant information, we  might want to consider implementing a filter by `dataset_id`? This way, we can streamline the process by directly accessing the data that's most pertinent.

Reply:
This is a good point. Shall we create a new story for data retrieval optimization? @emmambd @davidgamez